ChaosMod/Components/EffectDispatcher.h

```diff
@@ -64,6 +64,12 @@ class EffectDispatcher : public Component
 		}
 	};
 
+	enum TravelledDistanceType
```

Comment:
Would be cleaner to use enum class here to not pollute the global namespace

Reply:
👌